tests/login.go

```diff
@@ -60,28 +53,28 @@ func LoggedInCirrosExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 
 	if err != nil {
 		log.DefaultLogger().Object(vmi).Infof("Login: %v", resp)
-		expecter.Close()
-		return nil, err
+		return err
 	}
 
 	err = configureConsole(expecter, true)
 	if err != nil {
-		expecter.Close()
-		return nil, err
+		return err
 	}
 
-	return expecter, configureIPv6OnVMI(vmi, expecter, virtClient)
+	err = configureIPv6OnVMI(vmi, expecter, virtClient)
```

Comment:
We can just inline it with return since we are not wrapping the error or doing any logging.

Reply:
No problem, we are going to refactor it on the move to console PR